packages/gatsby-theme-newrelic/src/components/SEO.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+import { graphql, useStaticQuery } from 'gatsby';
+import useLocale from '../hooks/useLocale';
+const path = require('path');
+
+const SEO = ({ title, location }) => {
+  const {
+    site: { siteMetadata },
+    allLocale: { nodes: locales },
+  } = useStaticQuery(graphql`
+    query {
+      site {
+        siteMetadata {
+          defaultTitle: title
+        }
+      }
+      allLocale {
+        nodes {
+          locale
+          localizedPath
+        }
+      }
+    }
+  `);
+
+  const currentLocale = useLocale();
+
+  const { defaultTitle } = siteMetadata;
+
+  const subPath =
+    currentLocale === 'en'
```

Comment:
Rather than hardcoding `en` here, could you instead query for `isDefault` and find the default locale? While we are using `en` right now, if this value ever changes, its going to be tough to grep for. By finding the default locale, we can ensure that we are always using the configured value, which makes changing the default easy. 

```js
useStaticQuery(graphql`
  allLocale {
    nodes {
      locale
      localizedPath
      isDefault
  }
`)

const defaultLocale = locales.find(({ isDefault }) => isDefault)

currentLocale.locale === defaultLocale.locale?
```

Reply:
On that note, the `curentLocale` you pull in here is a full `Locale` node. You'll need to compare `currentLocale.locale` to ensure you're comparing string values.

Reply:
yes! will do that, and thank you for pointing that out. I will make that change